lib/libs/email/content/email-components.tsx

```diff
@@ -73,7 +73,7 @@ const LoginInstructions = ({
   <ul style={{ marginLeft: "-20px" }}>
     <li>
       <Text style={styles.text.description}>
-        The submission can be accessed in the OneMAC application at{" "}
+        The submission can be accessed in the OneMAC application, which you can find at{" "}
```

Comment:
the language was changed for ALL email templates for this one so I felt comfortable changing this component